test/index.html

```diff
@@ -56,14 +56,16 @@
                 }
 
                 /* global BoxAnnotations */
-                var annotations = new BoxAnnotations(null, { features: { drawing: true } });
+                var annotations = new BoxAnnotations(null, { features: { discoverability: true, drawing: true } });
```

Comment:
Do we plan to remove the feature flip and options in the near future?

Reply:
Good question, there are still exclusions but I can follow up with product